static/css/custom.css

```
@@ -534,3 +534,29 @@ because the nesting selector cannot represent pseudo-elements. */
         margin: 16px auto;
     }
 }
+
+
+/* allauth stuff */
+
+.allauth h1 {
+    font-size: 2rem;
+    text-weight: bold;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)